tests/director/tests/formation_constraint_api_test.go

```diff
@@ -16,16 +14,20 @@ import (
 func TestCreateFormationConstraint(t *testing.T) {
 	// GIVEN
 	ctx := context.Background()
+	priority := 7
```

Comment:
Do you think it's a good idea to add the `priority` and `description` fields in the update test (`TestUpdateFormationConstraint`) as well?